lib/WTSI/NPG/HTS/TreePublisher.pm

```diff
@@ -80,6 +91,10 @@ has 'require_checksum_cache' =>
                Function returning true for each file path to be published.
                CodeRef. Optional.
 
+               mlwh_json_cb
+               Callback writing information of a collection to JSON file.
```

Comment:
"writing information about data objects to a JSON file"

Reply:
Corrected